Sources/Gherkin/Models.swift

```diff
@@ -97,13 +109,15 @@ public struct ScenarioSimple: Codable {
 
 /// A gherkin Scenario Outline: with at least one example.
 public struct ScenarioOutline: Codable {
+    public var tags: [Tag]
```

Comment:
I’d prefer tags to be an optional array. The parser should ensure that tags is never accidentally an empty array.